### PR TITLE
Adding my photo and credential to the website

### PR DESCRIPTION
**Pull request by alejoheredia**

### CODE REVIEW

**Comment by alejoheredia** [comment]
Preview: https://heredroky.github.io/gci16.fossasia.org/index.html

**Comment by hemantjadon** [comment]
Just resolve the conflicts and you are good to go.

**Comment by alejoheredia** [comment]
Which are the conflicts?

**Comment by hemantjadon** [comment]
Merge conflicts

**Comment by alejoheredia** [comment]
Ok, but please wait a minute

**Comment by alejoheredia** [comment]
I think it is ready

**Comment by hemantjadon** [comment]
Okay this seems pretty good to me.

But the blog section made by @rhemon seems not to be working. Please check. Then you are good to go.

**Comment by alejoheredia** [comment]
I've checked many times what's happening with that, but I don't what's causing that problem :(

File: index.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
----
```
Comment (hemantjadon) [inline index.html:1]: You have removed these dashes which are important for jekyll add them and things will work out

Comment (alejoheredia) [reply index.html:1]: ok, I will correct it, thank you :)

Comment (alejoheredia) [reply index.html:1]: No, it isn't working yet :/
https://heredroky.github.io/gci16.fossasia.org/index.html

Comment (Heredroky) [inline index.html]: But I will try to fix that

**Comment by rhemon** [comment]
@heredroky at the top of the html to work jekyll you need to have 
---
---
note: they must be on two separate lines, you have only added once. 
hope it helps :)

**Comment by rhemon** [comment]
sorry it should be this
"---"

"---"
in previous comment it seems it changed without comment (dont add the double quote)

**Comment by alejoheredia** [comment]
Do you mean something like this?

---
---

**Comment by alejoheredia** [comment]
"---"
"---"

**Comment by rhemon** [comment]
yes

**Comment by alejoheredia** [comment]
Thank you Thank you Thank you Thank you Thank you Thank you Thank you :D It worked!!!
Now I need that someone merges that

**Comment by rhemon** [comment]
haha. no problems. congrats :)

**Review by tradzik — CHANGES_REQUESTED** [review]
Commits should be [squashed](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) before we can merge it. Adding 11 commits with the same purpose causes mess.

**Pull request closed without merging**